platform/broadcom/sonic-platform-modules-cel/haliburton/scripts/cpu_wdt

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python
+
+import os
+import time
+import signal
+import syslog
+import argparse
+import subprocess
+
+from sonic_platform_base.watchdog_base import WatchdogBase
+
+SYSLOG_IDENTIFIER = 'cpu_wdt'
+CPUWDT_MAIN_TASK_RUNNING_FLAG = True
+
+PLATFORM_CPLD_PATH = '/sys/devices/platform/e1031.smc/'
```

Comment:
how do we make sure the platform driver is installed and working when this service is started?

Reply:
I have added platform-modules-haliburton as the required service in the cpu_wdt service.
And also added a check for driver files to the script https://github.com/Azure/sonic-buildimage/pull/6259/commits/c5344ce50a9a1371e6a06b34d7c880f3fbc34510

Reply:
since it might be a timing issue, instead exit there, it would be better if you can poll at 1 second interval for 60 seconds, and check if the file exists or not, if the file still does not exist after 60 seconds, then exit and generating an error.

Reply:
OK, added in https://github.com/Azure/sonic-buildimage/pull/6259/commits/0ad2f6cac81cbeeeae1a0ada9b60686a51009d8f